apps/backend/src/app/hotel/hotel.module.ts

```diff
@@ -1,9 +1,11 @@
 import { Module } from '@nestjs/common';
 import { HotelController } from './hotel.controller';
 import { HotelService } from './hotel.service';
+import { CloudinaryService } from '../cloudinary/cloudinary.service';
+import { ConfigService } from '@nestjs/config';
 
 @Module({
   controllers: [HotelController],
-  providers: [HotelService],
+  providers: [HotelService, CloudinaryService, ConfigService],
```

Comment:
CloudinaryService should provide om CloudinaryModule

Reply:
Because in here I'm using hotelController for both HotelService and CloundinaryService ( just 2 methods update and delete image, so I'm not create a new controllers for this service). That why the providers of HotelController must declared CloudinaryService.